liferay-echo/liferay-portal
# LPS-197153 Fix Picker focus after changing selections and improvements in the text for screen readers

Motivation
=======
Focus was lost when selecting an options in the page editor language selector. Also, the sr message on each option should be updated from Language en-US: Default to en-US Language: Default .
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-197153)

Steps to Verify:
----------------
1. Open the page editor
1. Select a language using the keyboard
1. Use a screen reader to check that the new text is read. 


Screenshots:
-----------------------
<img width="397" alt="Captura de pantalla 2023-09-25 a las 16 27 47" src="https://github.com/liferay-echo/liferay-portal/assets/3276218/14b86331-1093-449c-9d11-a9072ac4d9fd">
